### PR TITLE
Add Desert Financial Credit Union and Lockheed Georgia Employees Community Credit Union

### LINKED ISSUE #2473: Please add Desert Schools Federal Credit Union.
Thank you!

### PR DESCRIPTION
**Pull request by brycx**
Fixes #2473 and #2411 

Desert Schools Federal CU has since the creation of issue #2473 [changed their name to "Desert Financial Credit Union"](https://www.desertfinancial.com/newname).

### CODE REVIEW

**Review by Carlgo11 — DISMISSED** [review]
I suggest this image instead: dfcu

Also, I didn't find anything mentioning sms or email 2fa on the doc link for Desert Financial Credit Union. Did I miss something?

**Comment by Carlgo11** [comment]
Look at that https://www.ssllabs.com/ssltest/analyze.html?d=desertfinancial.com 😄 
Wouldn't want to have my money there... 😅

**Comment by mxxcon** [comment]
wow..🤦‍♂️

**Comment by brycx** [comment]
@Carlgo11 This is from the link: 

> Multifactor Authentication
Desert Financial confirms your identity via at least two unique elements: ATM transactions that require a separate card and PIN, and Online Banking, which requires a short code verification sent to your phone or email.

It is the only thing I could find then and now. I assume the actual docs for it, is only available to registered customers. But by that sslabs score, I don't think it makes much of a difference...

File: _data/banking.yml [hunk 1]
```diff
@@ -500,6 +508,13 @@ websites:
       img: lendingclub.png
       tfa: No
 
+    - name: LGE Community Credit Union
+      url: https://www.lgeccu.org/
+      twitter: LGEccu
+      facebook: LGECommunityCreditUnion
```
Comment (Carlgo11) [inline _data/banking.yml]: Their Facebook page seems to be deleted. Remove it and the PR is good to go.
```suggestion
```

**Review by Carlgo11 — DISMISSED** [review]

**Review by Carlgo11 — DISMISSED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Review by kmpoppe — APPROVED** [review]

**Pull request merged**